src/Microsoft.Azure.ServiceBus/MessageReceivePump.cs

```diff
@@ -176,7 +179,7 @@ void CancelAutoRenewlock(object state)
             }
             catch (Exception exception)
             {
-                this.registerHandlerOptions.RaiseExceptionReceived(new ExceptionReceivedEventArgs(exception, ExceptionReceivedEventArgsAction.Complete));
+                this.registerHandlerOptions.RaiseExceptionReceived(new ExceptionReceivedEventArgs(exception, ExceptionReceivedEventArgsAction.Complete, namespaceName));
```

Comment:
I really think `this.` is a noise and wish we'd move away from it 😞
But for this PR will do.